### ISSUE TITLE
Sharing of workfields and woodcutting can be improved

### ISSUE THREAD

**Issue by TehPlayer14** [Kromster80/kam_remake#44]
Workers could own fields/trees when they are coming instead of when they are at it. This could potentially make their work more efficient

Affects:
Woodcutter's
Farm
Vineyard

**Comment by Kromster80** [comment]
I don't think this will make a huge difference, because in KMR workers who cannot reach their planned field will try to pick another field on contact.
